### PR TITLE
[CN-309] Add ImagePullSecrets into Hazelcast and MC CRs

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

**Review by hakanmemisoglu — APPROVED** [review]

File: bundle.yaml [hunk 1]
```diff
@@ -73,6 +73,16 @@ spec:
                 default: IfNotPresent
                 description: Pull policy for the Hazelcast Platform image
                 type: string
+              imagePullSecrets:
+                description: Image pull secrets for the Hazelcast Platform image
+                items:
+                  description: LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
+                  properties:
+                    name:
+                      description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names TODO: Add other useful fields. apiVersion, kind, uid?'
```
Comment (hasancelik) [inline bundle.yaml:82]: Is TODO left-over?

Comment (hakanmemisoglu) [reply bundle.yaml:82]: I think it is coming from Operator SDK generation since it is using the source code from Kubernetes API. TODO must be there.

Comment (mtyazici) [reply bundle.yaml:82]: Exactly

Comment (hasancelik) [reply bundle.yaml:82]: I see, thanks for the explanation 👍

**Review by hasancelik — APPROVED** [review]

**Pull request merged**